### ISSUE TITLE
fs.indexed_chars seems to also affect the REST API

### ISSUE THREAD

**Issue by shadiakiki1986** [dadoonet/fscrawler#491]
My understanding from the documentation was that `fs.indexed_chars` only affects fscrawler when indexing the local filesystem. However, I just found that when uploading a file via REST API, the uploaded file is also truncated to `100,000` by default.

Reading through the README, the `fs.indexed_chars` is alongside things like 

```
{
  "name" : "job_name",
  "fs" : {
    "url" : "/path/to/data/dir",
    "update_rate" : "15m"
  }
}
```

Maybe you need to split out options that are common to crawling the filesystem and to REST?

e.g.

```
...
"fs": {
  "url": ...
  "update_rate": ...
},
"rest": {
  ...
},
# New section common to uploading files via REST or Filesystem
"processing": {
  ...
}
```

Edit: Or maybe just add a note in the documentation about this and just call it a `feature` :1st_place_medal:  :smile:

**Comment by dadoonet** [comment]
Yes. That's totally the plan to revisit all the configuration settings when I'll work on #268.

It's kind of mess for now. I agree with that. In the meantime, would you like to add some comments in the REST part documentation stating that most of the fs settings are also used by the REST API?

**Comment by dadoonet** [comment]
As the documentation has been updated with #492 and that I'm planning to "fix" that with #268, I'm closing the current issue if you don't mind.
